torchaudio/functional/functional.py

```diff
@@ -1318,9 +1320,15 @@ def _get_sinc_resample_kernel(
             "https://github.com/pytorch/audio/issues/1487."
         )
 
+    if resampling_method not in ['sinc_interpolation', 'kaiser_window']:
+        raise ValueError('Invalid resampling method: {}'.format(resampling_method))
+
     orig_freq = int(orig_freq) // gcd
     new_freq = int(new_freq) // gcd
 
+    if resampling_method == "kaiser_window" and beta is None:
+        beta = 6.
```

Comment:
my personal preference for default beta value is `kaiser_fast` because it feels balanced, but it's subjective and I am fine with 6 as well.